plugins/glustershd/transactions.go

```diff
@@ -0,0 +1,112 @@
+package glustershd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/gluster/glusterd2/glusterd2/brick"
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/servers/sunrpc/dict"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+
+	log "github.com/sirupsen/logrus"
+)
+
```

Comment:
It'll be good to get these 3 functions reviewed by someone who works on AFR.

@itisravi